daeun/모스부호-(1).js

```diff
@@ -0,0 +1,11 @@
+function solution(letter) {
+    let morse = { 
+    '.-':'a','-...':'b','-.-.':'c','-..':'d','.':'e','..-.':'f',
+    '--.':'g','....':'h','..':'i','.---':'j','-.-':'k','.-..':'l',
+    '--':'m','-.':'n','---':'o','.--.':'p','--.-':'q','.-.':'r',
+    '...':'s','-':'t','..-':'u','...-':'v','.--':'w','-..-':'x',
+    '-.--':'y','--..':'z'
+    }
+    
+    return letter.split(' ').map(item => morse[item]).join('')
+}
```

Comment:
```suggestion
function solution(letter) {
    const morse = { 
    '.-':'a','-...':'b','-.-.':'c','-..':'d','.':'e','..-.':'f',
    '--.':'g','....':'h','..':'i','.---':'j','-.-':'k','.-..':'l',
    '--':'m','-.':'n','---':'o','.--.':'p','--.-':'q','.-.':'r',
    '...':'s','-':'t','..-':'u','...-':'v','.--':'w','-..-':'x',
    '-.--':'y','--..':'z'
    }
    
    return letter.split(' ').map(item => morse[item]).join('')
}
```
- 오 굉장히 빨리 푸셨네요 👍
- morse 객체는 재할당이 이루어지지 않으므로 `const`가 맞는 변수키워드인거 같습니다🙆‍♂️

Reply:
@yunwoo-yu const와 let은 계속 헷갈리네요 🥲 감사합니다!!